mteb/load_results/task_results.py

```diff
@@ -511,8 +511,15 @@ def validate_and_filter_scores(self, task: AbsTask | None = None) -> AbsTask:
                 new_scores[split].append(_scores)
                 seen_subsets.add(_scores["hf_subset"])
             if seen_subsets != hf_subsets:
+                missing_subsets = hf_subsets - seen_subsets
+                if len(missing_subsets) > 2:
+                    subset1, subset2 = list(missing_subsets)[:2]
+                    missing_subsets_str = f"{{'{subset1}', '{subset2}', ...}}"
```

Comment:
[Minor] I worry that users may not notice that we had limited to the 1st 2 missing subsets, and assume that there were only 2 from the current log message. I wonder if mentioning this fact explicitly would help. (We only knew because of the PR description and had read the relevant code here)

```suggestion
                    missing_subsets_str = f"First two: {{'{subset1}', '{subset2}', ...}}"
```